packages/console/src/pages/Connectors/components/Guide/index.tsx

```diff
@@ -58,6 +61,14 @@ const Guide = ({ connector, onClose }: Props) => {
     watch,
   } = methods;
 
+  useEffect(() => {
+    const { isStandard } = connector;
+
+    if (isStandard) {
+      setCallbackConnectorId(generateStandardId());
+    }
```

Comment:
Should we check `platform` and `setCallbackConnectorId` when it's a social connector?

Reply:
Agree.

Reply:
updated